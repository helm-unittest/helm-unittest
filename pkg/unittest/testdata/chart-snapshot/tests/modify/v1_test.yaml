suite: Test with simple snapshot modification
templates:
  - templates/test.yaml
tests:
  - it: should render correctly for test environment
    asserts:
      - matchSnapshot: {}
