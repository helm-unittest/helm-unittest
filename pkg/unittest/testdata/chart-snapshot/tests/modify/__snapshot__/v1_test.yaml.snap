should render correctly for test environment:
  1: |
    foo: bar
